modules/prefs.js

```diff
@@ -65,6 +65,7 @@ function PrefManager() {
   this.hide_quote_length = prefsService.getIntPref("hide_quote_length");
   this.hide_sigs = prefsService.getBoolPref("hide_sigs");
   this.compose_in_tab = prefsService.getBoolPref("compose_in_tab");
+  this.message_open_in_conversation = prefsService.getBoolPref("message_open_in_conversation");
```

Comment:
I omitted the watch for changes part because this is a monkeypatch, I'd have to write out the code to undo the monkeypatch if we're doing in-process change (right now the doc indicates restart required).     If you're interested in the patch I can take a stab at that.